crates/iota-json-rpc-types/src/iota_transaction.rs

```diff
@@ -2366,6 +2366,10 @@ pub enum TransactionFilter {
     /// Query by changed object, including created, mutated and unwrapped
     /// objects.
     ChangedObject(ObjectID),
+    /// Query transactions that wrapped or deleted the specified object.
+    /// Includes transactions that either created and immediately wrapped
+    /// the object or unwrapped and immediately deleted it.
+    WrappedOrDeletedObject(ObjectID),
```

Comment:
This isn't backward compatible either for the Rust API.

Reply:
Fixed by https://github.com/iotaledger/iota/pull/7133/commits/73e8cc935ecfd0c0781d55eb6ef51271430e7aee